server/controllers/events/events_controller_e2e_test.go

```diff
@@ -960,10 +989,15 @@ func setupE2E(t *testing.T, repoDir, repoConfigFile string) (events_controllers.
 		GitlabUser:  "gitlab-user",
 		GitlabToken: "gitlab-token",
 	}
+	allowCommands := []command.Name{command.Plan, command.Apply, command.Unlock, command.ApprovePolicies, command.Import}
```

Comment:
Can we make this struct delimited by newlines since we'll probably append more items here?